pypeit/pypeit.py

```diff
@@ -295,33 +330,48 @@ def reduce_exposure(self, frames, bg_frames=[], std_outfile=None):
         Args:
             frame (:obj:`int`):
                 0-indexed row in :attr:`fitstbl` with the frame to
-                reduce
-            bgframes (:obj:`list`, optional):
-                List of frame indices for the background
+                reduce.
+            bg_frames (:obj:`list`, optional):
+                List of frame indices for the background.
             std_outfile (:obj:`str`, optional):
-                the name of a file with a previously PypeIt-reduced standard spectrum.
+                File with a previously reduced standard spectrum from
+                PypeIt.
 
         Returns:
-            dict: The dictionary containing the primary outputs of extraction
+            dict: The dictionary containing the primary outputs of
+            extraction.
+
         """
+
+        # TODO:
+        # - bg_frames should be None by default
+        # - change doc string to reflect that more than one frame can be
+        #   provided
+
```

Comment:
the codes require that rather than None it be an empty list. we can debate this, but trying to make it None will break a bunch of stuff.

Reply:
I dislike empty lists as a default.  Leaving it for now and leaving the comment.